packages/backend/src/connector/core/middleware/balance.ts

```diff
@@ -45,7 +45,9 @@ export function createBalanceMiddleware(): ILPMiddleware {
     }
 
     // Update balances on prepare
-    const createPendingTransfer = async (): Promise<Transaction> => {
+    const createPendingTransfer = async (): Promise<
+      Transaction | undefined
+    > => {
```

Comment:
```suggestion
    const createPendingTransfer = async (): Promise<Transaction> => {
```
It looks like we can instead do `return ctxThrow` below:
https://github.com/interledger/rafiki/blob/421d89ac7883cec8d0f9f5a0d31cda98b6a9a448/packages/backend/src/connector/core/middleware/balance.ts#L64

Reply:
But `ctx.Throw()` doesn't throw so that block returns `undefined`.